react/src/types/projects.ts

```diff
@@ -0,0 +1,17 @@
+export interface Project {
+  description: string;
+  id?: number;
+  name: string;
+  ds_id?: string;
+  title?: string;
+  uuid?: string;
+  public?: boolean;
+  system_file?: string;
+  system_id?: string;
+  system_path?: string;
+  deletable?: boolean;
+  deleting?: boolean;
+  deletingFailed?: boolean;
```

Comment:
I think these two can be dropped.  They aren't part of the response from the backend;  I think they are added on the angular side but won't be needed in react.
```suggestion
```